src/d3a/models/strategy/home_meter.py

```diff
@@ -189,6 +190,20 @@ def event_market_cycle(self):
 
         self._delete_past_state()
 
+    def set_energy_measurement_of_last_market(self):
```

Comment:
If this method is not supposed to be called from the outside, do you think we could mark it as private?

Reply:
This was introduced in my PR, but I think it could be changed. Anyway, not major.

Reply:
sorry, forgot to remove it (when I changed to your branch as target).

Reply:
sorry, again not read correctly. I'll make it private. Thanks!